test/testdrive/catalog.td

```diff
@@ -54,6 +54,18 @@ id          database
 -----------------------
 1           materialize
 
+# SHOW DATABASES should work, even if the current database has been dropped.
+> CREATE DATABASE j
+
+> SET DATABASE = j
+
+> DROP DATABASE j
+
+> SHOW DATABASES
+database
+----
+materialize
+
```

Comment:
Isn't this test case exactly the same as the test on L131? Doesn't seem like we need to test it twice!